app/controllers/sessions_controller.rb

```diff
@@ -9,7 +9,11 @@ def create
     user.save
     session[:user_id] = user.id
     flash[:notice] = "Logged in as #{user.name}"
-    redirect_to '/'
+    if user.admin?
+      redirect_to '/admin/dashboard'
+    else
+      redirect_to '/user/dashboard'
+    end
```

Comment:
This is so the dashboards can handle whether or not a visitor is trying to save an NPC by logging in